css-view-transitions-2/Overview.bs

```diff
@@ -450,6 +450,11 @@ Note: as per default behavior, the ''@view-transition'' rule can be nested insid
 	Note: the [=@view-transition/type=] descriptor only applies to the {{Document}} in which it is defined.
 	The author is responsible for using their chosen set of types in both documents.
 
+## Responding to changes in the ''@view-transition'' rule ## {#respond-to-rule-changes}
+
+When the ''@view-transition'' rule changes for {{Document}} |document|, [=update the opt-in state for outbound transitions=] given |document|.
```

Comment:
Should we leave a note here to the effect of, "The UA is responsible for deciding when style resolution is executed to keep the opt-in state in sync with style mutations".

Reply:
I don't think that's necessary.